<launch>
  
  <arg name="meshIntopic" default="/frame_meshs"/>
  <arg name="meshtestclouds" default="/test_clouds"/>
  

  <!-- Launch the husky goal package -->
  <node pkg="mesh_map" type="mesh_map" name="husky_mesh_map" output="screen" >
    <param name="mesh_in_topic" type="string" value="$(arg meshIntopic)" />
    <param name="map_test_clouds" type="string" value="$(arg meshtestclouds)" />     
  </node>


</launch>